src/scripts/towns/Town.ts

```diff
@@ -57,8 +57,8 @@ class Town {
 class DungeonTown extends Town {
     dungeon: Dungeon
 
-    constructor(name: string, region: GameConstants.Region, requirements: (Requirement | OneFromManyRequirement)[] = []) {
-        super(name, region, [], { requirements });
+    constructor(name: string, region: GameConstants.Region, content:  TownContent[] = [], requirements: (Requirement | OneFromManyRequirement)[] = []) {
```

Comment:
If you move the content-parameter to the end of the list, you won't need all the changes in TownList.ts

Reply:
Ah, thanks, that makes it a lot cleaner